easybuild/easyconfigs/o/OpenKIM/OpenKIM-1.8.2-foss-2016b.eb

```diff
@@ -0,0 +1,256 @@
+    'MO_636041334617_001.tgz',  # Pair_Morse_Shifted_GirifalcoWeizer_MedCutoff_Na
+    'MO_758825945924_001.tgz',  # Pair_Morse_Shifted_GirifalcoWeizer_MedCutoff_Ni
+    'MO_958424213898_001.tgz',  # Pair_Morse_Shifted_GirifalcoWeizer_MedCutoff_Pb
+    'MO_147245690895_001.tgz',  # Pair_Morse_Shifted_GirifalcoWeizer_MedCutoff_Rb
+    'MO_390128289865_001.tgz',  # Pair_Morse_Shifted_GirifalcoWeizer_MedCutoff_W
+    'MO_169434419764_001.tgz',  # Pair_Morse_Shifted_Glyde_Ne
+    'MO_831902330215_001.tgz',  # Pair_Morse_Shifted_Jelinek_Ar
+    'MO_071460865933_000.tgz',  # Pair_Morse_Sigmoid_Jelinek_Ar
+    'MO_908645784389_000.tgz',  # Pair_Morse_Smoothed90_Jelinek_Ar
+    'MO_500121566391_000.tgz',  # Tersoff_LAMMPS_Albe_Nordlund_Averback_PtC
+    'MO_799020228312_000.tgz',  # Tersoff_LAMMPS_Albe_Nordlund_Nord_Kuronen_GaAs
+    'MO_903987585848_001.tgz',  # Tersoff_LAMMPS_Erhart_Albe_CSi
+    'MO_408791041969_000.tgz',  # Tersoff_LAMMPS_Erhart_Albe_SiII_CSi
+    'MO_616776018688_000.tgz',  # Tersoff_LAMMPS_Erhart_Juslin_Goy_Nordlund_Mueller_Albe_ZnO
+    'MO_137964310702_000.tgz',  # Tersoff_LAMMPS_Mueller_Erhart_Albe_Fe
+    'MO_612061685362_000.tgz',  # Tersoff_LAMMPS_Nord_Albe_Erhart_Nordlund_GaN
+    'MO_245095684871_000.tgz',  # Tersoff_LAMMPS_Tersoff_PRB37_1988_Si
+    'MO_186459956893_000.tgz',  # Tersoff_LAMMPS_Tersoff_PRB38_1988_Si
+    'MO_171585019474_000.tgz',  # Tersoff_LAMMPS_Tersoff_PRB39_1989_CSi
+    'MO_350526375143_000.tgz',  # Tersoff_LAMMPS_Tersoff_PRB39_1989_GeSi
+    'MO_579868029681_000.tgz',  # Tersoff_LAMMPS_Tersoff_PRL61_1988_C
+    'MO_113686039439_002.tgz',  # Three_Body_Stillinger_Weber_Balamane_Si
+    'MO_503261197030_000.tgz',  # Three_Body_Stillinger_Weber_CdTeZnSeHgS
+    'MO_119167353542_002.tgz',  # Three_Body_Stillinger_Weber_Hauch_Brittle_Si
+    'MO_405512056662_003.tgz',  # Three_Body_Stillinger_Weber_Si
+    'MO_800412945727_002.tgz',  # Three_Body_Stillinger_Weber_Zhang_Silicene_Model1_Si
+    'MO_475612090600_002.tgz',  # Three_Body_Stillinger_Weber_Zhang_Silicene_Model2_Si
+    'MO_154399806462_001.tgz',  # model_ArCHHeXe_BOP_AIREBO
+    'MO_135034229282_001.tgz',  # model_Fe_PF_DudarevDerlet
+    'MO_140444321607_001.tgz',  # model_Fe_PF_chiesa_quinticsplines
+    'MO_856295952425_001.tgz',  # model_Fe_PF_mendelev
+    'MO_126942667206_001.tgz',  # model_He_P_AbIn
+    'MO_424746498193_001.tgz',  # model_Mo_PF_cubicsplines
+    'MO_218026715338_001.tgz',  # model_Nb_PF_cubicsplines
+    'MO_261274272789_001.tgz',  # model_Ta_PF_cubicsplines_derletnguyenmanhdudarev
+    'MO_683890323730_001.tgz',  # model_V_PF_cubicsplines_derletnguyenmanhdudarev
+    'MO_195478838873_001.tgz',  # model_W_PF_cubicsplines
+]
+
+
+# Now the sources list should be changed, so that models and model drivers
+# are unpacked in the right folders.
+
+kim_maindir = 'kim-api-v%s' % (version,)
+kim_subdir = {'MO': 'models',  'MD': 'model_drivers'}
+for i, x in enumerate(sources):
+    if x.startswith('MO_') or x.startswith('MD_'):
```

Comment:
@schiotz I don't think this is something we want in an easyconfig file, anything that requires executing code should go into an easyblock...

Since the pattern is fairly obvious, how about doing something like this?

```python
mo_subdir = 'models'
md_subdir = 'model_drivers'
sources = [
    'kim-api-v%(version)s.tgz',
    #
    # Model drivers
    #
    {
        'filename': 'MD_853402641673_000.tgz',  # EAM_CubicNaturalSpline
        'extract_cmd': "tar xvz kim-api-v%%(version)s/src/%s -f %%s" % md_subdir,
    }
    ...
```

That would still be a nightmare to maintain though, it's a lot easier to just specify a 2 lists with `MD_*.tgz` and `MO_*.tgz` filenames, and then let a custom OpenKIM easyblock loop over them to extract them in the right place?